### DISCUSSION TITLE
Setting static IP for WiCAN

### DISCUSSION THREAD

**Discussion by olterion** [meatpiHQ/wican-fw#268] [Q&A]
Hello,

is there any possibility to set a static IP address of the WiCAN OBD device?

Best regards

**Comment by meatpiHQ** [comment]
@olterion why not use mDNS?

**Reply by timtucker-dte** [reply]
Connection times are usually just a hair faster when IP / DNS are static.

**Reply by meatpiHQ** [reply]
It's on the list of improvements

**Comment by micha53** [comment]
I can tell to my router (Fritz!Box) that it shall always assign the same IP-Address to a device.
